[2023-12-20T11:48:41.334+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.second_id scheduled__2021-12-14T00:00:00+00:00 [queued]>
[2023-12-20T11:48:41.351+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.second_id scheduled__2021-12-14T00:00:00+00:00 [queued]>
[2023-12-20T11:48:41.352+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 6
[2023-12-20T11:48:41.460+0000] {taskinstance.py:2192} INFO - Executing <Task(BashOperator): second_id> on 2021-12-14 00:00:00+00:00
[2023-12-20T11:48:41.475+0000] {standard_task_runner.py:60} INFO - Started process 5437 to run task
[2023-12-20T11:48:41.484+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'second_id', 'scheduled__2021-12-14T00:00:00+00:00', '--job-id', '1259', '--raw', '--subdir', 'DAGS_FOLDER/first_dat.py', '--cfg-path', '/tmp/tmplduazl04']
[2023-12-20T11:48:41.490+0000] {standard_task_runner.py:88} INFO - Job 1259: Subtask second_id
[2023-12-20T11:48:41.731+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag_v2.second_id scheduled__2021-12-14T00:00:00+00:00 [running]> on host 29b3723064ac
[2023-12-20T11:48:42.103+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abrham' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='second_id' AIRFLOW_CTX_EXECUTION_DATE='2021-12-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-12-14T00:00:00+00:00'
[2023-12-20T11:48:42.105+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-12-20T11:48:42.105+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will be runing after task 3']
[2023-12-20T11:48:42.157+0000] {subprocess.py:86} INFO - Output:
[2023-12-20T11:48:42.166+0000] {subprocess.py:93} INFO - hey, I am task2 and will be runing after task 3
[2023-12-20T11:48:42.167+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-12-20T11:48:42.234+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_id, execution_date=20211214T000000, start_date=20231220T114841, end_date=20231220T114842
[2023-12-20T11:48:42.353+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2023-12-20T11:48:42.389+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
